package.json

```diff
@@ -31,6 +31,7 @@
     "integration:token": "node -r ./server/script/preflight ./server/script/generateIntegrationToken.js",
     "detachDeletedRepos": "node -r ./server/script/preflight ./server/script/detachDeletedRepos.js",
     "migrateAssessmentPools": "node -r ./server/script/preflight ./server/script/migrateAssessmentPools.js",
+    "migrateAssetsLocation": "node -r ./server/script/preflight ./server/script/migrateAssetsLocation.js",
```

Comment:
Since you are adding another one script, `preflight` command logic is same in all scripts above. It would be nice thing to separate it as `preflight` script and then reuse it in where it's needed. :bowtie:

Reply:
That's not possible since `-r ./server/script/preflight` is a runtime executable argument - [check the docs](https://nodejs.org/api/cli.html#cli_r_require_module). 
If you split it to `node -r ./server/script/preflight` and `node [script-name.js]` you would get one node REPL with `preflight` prerequired and script run without `preflight`.